### DISCUSSION TITLE
海外版PCL可能需要考虑

### DISCUSSION THREAD

**Discussion by Crotina** [Meloong-Git/PCL#4816] [讨论]
我的那几个希腊同学还在用3代i5，再看那些学校电脑年纪都比我大了哈哈哈哈哈哈哈哈，PCL可能只需要保留主要功能就够了，优化可能要多做一下🙃
或者可以单独出一个Lite版🙃🙃

**Comment by Joker2184** [comment]
问题不大（
PCL不吃性能
我试过四代i5和G2030 
比起优化启动器还不如玩优化好一点的版本或者低版本

**Comment by tangge233** [comment]
游戏比 PCL 还吃性能，应该多考虑怎么通过加模组等方式优化游戏体验
~~3代i5跑 MC 很困难吧……~~

**Reply by Crotina** [reply]
> 游戏比 PCL 还吃性能，应该多考虑怎么通过加模组等方式优化游戏体验 ~3代i5跑 MC 很困难吧……~

呃呃呃，我的意思是可以优化资源分配之类的，反正就那些

**Comment by Deep-Dark-Forest** [comment]
和 #2330 差不多
